build/AzurePipelinesTemplates/WindowsAppSDK-BuildProject-Steps.yml

```diff
@@ -4,8 +4,34 @@ parameters:
   buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
   artifactName: 'drop'
   channel: 'experimental'
+  enableLicenseInstall: false
+  # license installation assets are only accessible in internal builds, but are only
+  # needed in release-signed stable builds. Only enble this functionality in the private
+  # pipeline builds. Otherwise, defautl to an inert placeholder implementation.
+
 
 steps:
+  - ${{ if eq(parameters.enableLicenseInstall, true) }}:
+    - task: NuGetAuthenticate@0
+      inputs:
+        nuGetServiceConnections: 'ProjectReunionInternal'
+
+    # Restore transport package dependencies. This is only enbaled in release-signed builds.
+    - task: PowerShell@2
+      name: ConvertVersionDetailsToPackageConfig
+      displayName: "Convert VersionDetails To PackageConfig"
+      inputs:
+        filePath: '$(Build.SourcesDirectory)\build\Scripts\ConvertVersionDetailsToPackageConfig.ps1'
+        arguments: -versionDetailsPath '$(Build.SourcesDirectory)\eng\Version.Details.xml' -packageConfigPath '$(Build.SourcesDirectory)\build\packages.config'
+
+    - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+      displayName: RestoreNuGetPackages
+      inputs:
+        restoreSolution: build/packages.config
+        feedsToUse: config
+        nugetConfigPath: build/nuget.config
```

Comment:
Try using a different nuget.config path and remove this nuget.config?
https://dev.azure.com/ms/ProjectReunion/_build/results?buildId=242564&view=logs&j=d79de4a2-8ffa-524a-a762-f0c23acfce7b&t=f756dbec-87ac-58c9-915e-0f37668f1339&s=96ac2280-8cb4-5df5-99de-dd2da759617d
This task references the internal feed somehow when it shouldn't and I have a feeling the nuget.config in the build directory is causing this. There should be a nuget.config in the eng/common directory you can use.

Reply:
renamed the config file to avoid ambiguity. Nuget task takes it explicitly so no need to use a 'default' name.